templates/admin/forms/list.html.twig

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <style type="text/css">
+        {# Specific override to allow larger width on this container#}
+        .container {
+            width: 1500px;
+        }
+        .bottom-pad {
+            padding-bottom: 25px;
+        }
+    </style>
+    <title>{{ "Forms Administration"|translate }}</title>
```

Comment:
Lets just stick with one; either translate or the alias(choose one now and go with it). This will make collecting of the constants much simpler.